autopush-common/src/db/bigtable/mod.rs

```diff
@@ -48,6 +48,10 @@ pub struct BigTableDbSettings {
     /// bigtable.
     #[serde(default)]
     pub table_name: String,
+    /// Routing replication profile id.
+    /// Should be used everywhere we set `table_name` when creating requests
+    #[serde(default)]
+    pub profile_id: String,
```

Comment:
Could we rename this to `app_profile_id` (Bigtable docs seem to always say  "app" along with profile). I think we can just do this with a `serde(rename = "app_profile_id")` if you want to save the shorter name for the internal code.

Reply:
I'm fine just changing the symbol name to `app_profile_id`. Might as well not make things confusing later.